projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/experiment/ExperimentalVariable.java

```diff
@@ -0,0 +1,51 @@
+package life.qbic.projectmanagement.domain.project.experiment;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+class ExperimentalVariable<T extends ExperimentalValue> {
+
+  private final List<T> levels;
+
+  private final String name;
+
+  @SafeVarargs
+  public ExperimentalVariable(String name, T... levels) {
+    this.name = name;
+    this.levels = Collections.unmodifiableList(Arrays.asList(levels));
```

Comment:
This allows for a creation of an experimental variable without any levels. I think at least one? or two? is required.